src/zope/traversing/api.py

```diff
@@ -122,17 +87,15 @@ def traverseName(obj, name, default=_marker, traversable=None, request=None):
 
 
 def getName(obj):
-    """Get the name an object was traversed via
+    """
+    Get the name an object was traversed via
```

Comment:
Let's add a period at the end of the sentence while we're cleaning things up?

Reply:
Good idea!